python/lsst/ip/diffim/makeKernelBasisList.py

```diff
@@ -260,7 +260,7 @@ def generateAlardLuptonBasisList(config, targetFwhmPix=None, referenceFwhmPix=No
         # http://iopscience.iop.org/0266-5611/26/8/085002  Equation 40
 
         # Use specializations for deconvolution
-        logger.info("Target psf fwhm is the greater, deconvolution mode")
+        logger.debug("Target psf fwhm is the greater, deconvolution mode")
```

Comment:
This particular deconvolution mode will never get triggered in GAaP by construction. And if at all it gets triggered (either due to a bug in GAaP or in diffim), I think we'd want to know immediately. So perhaps, this should/could stay at INFO level?